tools/misc/mapping_expander.py

```
@@ -0,0 +1,194 @@
+
+SCRIPT_VERSION = '0.2'
+
+import pandas as pd
+from openpyxl import Workbook
+from openpyxl.styles import Font, numbers
+from openpyxl.utils.dataframe import dataframe_to_rows
+import os
+import json
+
+# === Configuration variables ===
+source_file = "../excel/ccf/part_mapping_adobe-ccf-v5_to_soc2.xlsx"
```

Pylint (3.3.7)

[convention] Constant name "source_file" doesn't conform to UPPER_CASE naming style

(C0103)